app/controllers/user.js

```diff
@@ -0,0 +1,22 @@
+const User = require('../models').User,
+  logger = require('../logger'),
+  errors = require('../errors'),
+  bcryptService = require('../services/bcrypt');
+
+exports.signUp = ({ user }, res, next) => {
+  bcryptService
+    .hashPassword(user.password)
+    .then(hashed => {
+      user.password = hashed;
```

Comment:
You should not modify object `user`. Create a new object with the pass hashed or send to `User.create` the object deconstruct with the new key password

Reply:
Read of operator spread